### PR TITLE
[MODULAR] marked one fixes / more changes

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

This PR removes unused variables wthin the gladiator module var while adjusting the general balance of the marked one, buffing his movement speed and fixing dysfunctional charge attacks while also adding more obvious alerts to spin and charge attacks through voicelines rather than sounds or chat alerts.

In short:
-his third and fourth phases have faster movement speeds
-his charge speed is no longer a snail's pace
-his charge is no longer overridden by having another attack queued
-his charge and spin attacks make clang noises
-he talks when doing a spin or charge
-he has bubble alerts for things like blocking damage
-spin has more range
-shockwaves radiate faster
-his first stage block chance is now applied in his last stage
-he no longer floats as if he's in zero gravity
-unused variables in his equipment are gone :DD

## How This Contributes To The Skyrat Roleplay Experience

"roleplay" is a stretch, but this is more just me maintaining the things i added to the game, using player feedback and active round observation as a benchmark

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image

</details>

## Changelog

:cl:
qol: marked one is now harder and makes clang noises and talks to signal attacks
fix: removed useless berserker armor vars
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 1]
```diff
@@ -351,8 +373,10 @@
 /mob/living/simple_animal/hostile/megafauna/gladiator/proc/discharge(modifier = 1)
 	stunned = TRUE
 	charging = FALSE
+	balloon_alert_to_viewers("THUD")
```
Comment (Zonespace27) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: why

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 2]
```diff
@@ -199,12 +199,14 @@
...
 		deltimer(anger_timer_id)
 	anger_timer_id = addtimer(CALLBACK(src, PROC_REF(get_calm)), MARKED_ONE_ANGER_DURATION, TIMER_STOPPABLE)
+	balloon_alert_to_viewers("angered!")
 
 /// Makes the Marked One a sleepy boy that don't wanna hurt nobody. He starts like this and progresses to his hostile state after seeing an ash walker or being punched in the noggin.
 /mob/living/simple_animal/hostile/megafauna/gladiator/proc/get_calm()
 	if(anger_timer_id)
 		deltimer(anger_timer_id)
 	anger_timer_id = null
+	balloon_alert_to_viewers("docile")
```
Comment (Zonespace27) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: why

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm (hunk 2 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: good question

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm (hunk 1 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: good question

**Comment by dawsonkeyes** [comment]
since this is technically also balancing, a maintainer can edit tags so i'm not whoring GBP off of this xd

**Comment by dawsonkeyes** [comment]
wake the fuck up samurai

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**